### ISSUE TITLE
directus_users password required validation is too aggressive (relational)

### ISSUE THREAD

**Issue by christianrr** [directus/api#1180]
## Bug Report

When creating a M2M pointing to directus_users, a new entry in the related table cannot be stored due to a validation error on directus_users (password should not be blank).

However, since I want to create a new entry in the related table (message in my case), no validation rule of directus_users should apply. Apart from that the password of admin@example.com is not null.

Happens also on a clean checkout of latest version.

<img width="1074" alt="Annotation 2019-08-05 231640" src="https://user-images.githubusercontent.com/10274479/62495977-d6234c80-b7d7-11e9-84a2-ef68037f373c.png">

`{code: 4, message: "password: This value should not be blank.", class: "Directus\Validator\Exception\InvalidRequestException", file: "/var/www/directus/src/core/Directus/Services/AbstractService.php", line: 186}`

### Steps to reproduce

1. Create new table message
2. Create new M2M relation to directus_users (auto generate junction table)
3. Create a new message with at least one user selected and then save

<img width="510" alt="Annotation 2019-08-05 230401" src="https://user-images.githubusercontent.com/10274479/62495986-dcb1c400-b7d7-11e9-8979-df23dcf1ec65.png">

## Technical Details

Device: [Virtual Server]
OS: [Ubuntu 18.04]
Web Server: [Apache 2.4.29]
PHP Version: [7.2.19]
Database: [5.7.27]
API Version: [2.3.1]
APP Version: [7.8.0]

Client:
Device: [Desktop, Chrome 75.0.3770.142]
OS: [Windows 10 1809]

**Comment by rijkvanzanten** [comment]
@directus/api-team the API will always return the `password` error message, even when only submitting the primary key:

```json
{
    "users": [
        {
            "directus_users_id": 1
        }
    ]
}
```

or

```json
{
    "users": [
        {
            "directus_users_id": {
                "id": 1
            }
        }
    ]
}
```

The API shouldn't throw the validation error for `password`, as that field already contains a value.

**Comment by rijkvanzanten** [comment]
I feel like this is a similar problem as https://github.com/directus/api/issues/1136, but this time `password` is being required hardcoded

**Comment by bjgajjar** [comment]
Fixed in #1183
